lambdas/enums/metadata_report.py

```diff
@@ -11,6 +11,7 @@ class MetadataReport(StrEnum):
     Date = "Date"
     Timestamp = "Timestamp"
     ID = "ID"
+    RegisteredAtUploaderPractice = "RegisteredAtUploaderPractice"
 
     @staticmethod
     def list():
```

Comment:
This method is no longer being used anywhere in the codebase. it can be removed.

Reply:
This has been actioned by Steph